scripts/win32.bat

```diff
@@ -54,5 +56,14 @@ For Each objDrive In colDiskDrives
             Wscript.Echo ""
         Next
     Next
+    If foundLDs = 0 Then
+      Wscript.Echo "device: """ & Replace(objDrive.DeviceID, "\", "\\") & """"
+      Wscript.Echo "description: """ & objDrive.Caption & """"
+      Wscript.Echo "size: " & objDrive.Size
+      Wscript.Echo "mountpoint: Null"
+      Wscript.Echo "name: Null"
+      Wscript.Echo "system: False"
```

Comment:
If we can't infer the value, I rather default it to `True`, so we're on the safe side.